scripts/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+case $1 in
```

Comment:
Yes I was trying to isolate the environment stuff. The makefiles also don't support it well, in particular capturing and applying bash 'eval' for perl and lua.  Once the caching is behaving I will probably come back and revisit this part again.  